mobile/components/ChatInput/index.js

```diff
@@ -33,7 +33,7 @@ class ChatInput extends React.Component<Props, State> {
           onChangeText={this.onChangeText}
           placeholder="Your message here..."
           multiline
-          autoFocus
+          autoFocus={false}
```

Comment:
I'm pretty sure we want this to auto focus? No?

Reply:
No, I don't think so - it's super frustrating to open threads and have the keyboard come up and block anything like scrolling or tapping onto any links. Maybe we can build out a better heuristic for this in the future.

Reply:
SGTM